kodilla-patterns/src/test/java/com/kodilla/patterns/builder/bigmac/BigMacTestSuite.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.patterns.builder.bigmac;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BigMacTestSuite {
+    @Test
+    public void testNewBigMac(){
+        //Given
+        BigMac bigmac = new BigMac.BigMacBuilder()
+                .burger(2)
+                .sauce("standard")
+                .ingredient("lettuce")
+                .ingredient("bacon")
+                .ingredient("chili peppers")
+                .bun(true)
+                .build();
+        System.out.println(bigmac);
+        //When
+        String sauce = bigmac.getSauce();
+        int ingredient = bigmac.getIngredients().size();
+        int burger = bigmac.getBurgers();
+        boolean isSesame = bigmac.isBun();
```

Comment:
dlaczego isBun to isSezame ?

Reply:
bo dziwnie dla mnie jest w zadaniu napisane:D z tymi bułkami... dlatego czy z sezamem czy bez tak wywnioskowałem